jobs/riemann/templates/config/memory.clj.erb

```diff
@@ -0,0 +1,31 @@
+(info "Loading memory alerts")
+
+(defn memory-alerts [pd]
+  (info "Setting up memory alerts")
+  (where (service "memory/percent-used")
+    (smap
+      (fn [event]
+        ; set the event's state based on the value of the metric, and possibly the specific host it originated from
+        (assoc event :state
+          (cond
+          <% p("riemann.memory.overrides").each do |over| %>
+            (and (re-matches #"<%= over["host"] %>" (:host event)) (> (:metric event) <%= over["threshold"] %>)) "<%= over["state"] %>"
```

Comment:
Updated the docs; I chose `re-matches` specifically to avoid writing alerts that actually matched unexpected hosts with partial matches.  If we find that this isn't flexible enough in the future, we can update it in the future